services/lemmy/lemmy.service.js

```diff
@@ -0,0 +1,90 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService, InvalidParameter } from '../index.js'
+
+const queryParamSchema = Joi.object({
+  server_fqdn: Joi.string().hostname(),
+}).required()
+
+const lemmyCommunitySchema = Joi.object({
+  community_view: Joi.object({
+    counts: Joi.object({
+      subscribers: Joi.number().required(),
+      posts: Joi.number().required(),
+      comments: Joi.number().required(),
+    }).required(),
+  }).required(),
+}).required()
+
+export default class Lemmy extends BaseJsonService {
+  static category = 'social'
+
+  static route = {
+    base: 'lemmy',
+    pattern: ':community',
+    queryParamSchema,
+  }
+
+  static examples = [
+    {
+      title: 'Lemmy',
+      namedParams: { community: 'asklemmy@lemmy.ml' },
+      staticPreview: this.render({
+        community: 'asklemmy@lemmy.ml',
+        members: 42,
+      }),
+    },
+    {
+      title: 'Lemmy',
+      namedParams: { community: 'asklemmy@lemmy.ml' },
+      queryParams: { server_fqdn: 'lemmy.world' },
+      staticPreview: this.render({
+        community: 'asklemmy@lemmy.ml',
+        members: 42,
+      }),
+    },
+  ]
+
+  static _cacheLength = 30
+
+  static defaultBadgeData = { label: 'community' }
+
+  static render({ community, members }) {
+    return {
+      label: `subscribe to ${community}`,
+      message: `${metric(members)} subscribers`,
+      color: 'brightgreen',
```

Comment:
Normally the label (left side of the badge) should describe what the number in the message (right side means). Optionally with the social badges, this can be a call to action. Lets ditch the " subscribers" suffix from the message here.